src/nuget/Microsoft.DotNet.BuildTools.nuspec

```diff
@@ -15,30 +15,52 @@
     <copyright>Copyright © Microsoft Corporation</copyright>
     <tags></tags>
     <dependencies>
-     <!-- Do not add any dependencies to this package as we want to install it versionless (i.e. -ExcludeVersion) and we don't want the dependencies to be installed that way -->
+      <group targetFramework=".NETCoreApp1.0">
+        <dependency id="NuProj" version="[0.10.4-beta-gf7fc34e7d8, )" />
+        <dependency id="MSBuild" version="[0.1.0-preview-00024-160610, )" />
+        <dependency id="Microsoft.Build.Framework" version="[0.1.0-preview-00024-160610, )" />
+        <dependency id="Microsoft.Build.Tasks.Core" version="[0.1.0-preview-00024-160610, )" />
+        <dependency id="Microsoft.Build.Utilities.Core" version="[0.1.0-preview-00024-160610, )" />
+        <dependency id="Microsoft.Build.Targets" version="[0.1.0-preview-00024-160610, )" />
+        <dependency id="Microsoft.Build" version="[0.1.0-preview-00024-160610, )" />
+        <dependency id="Microsoft.Net.Compilers.NetCore" version="[1.3.0-beta1-20160429-01, )" />
+        <dependency id="Microsoft.Net.Compilers.Targets.NetCore" version="[0.1.5-dev, )" />
+        <dependency id="Microsoft.Cci" version="[4.0.0-rc3-24128-00, )" />
+        <dependency id="Microsoft.Composition" version="[1.0.30, )" />
+        <dependency id="System.Diagnostics.TextWriterTraceListener" version="[4.0.0-rc3-24128-00, )" />
+        <dependency id="System.Runtime.Serialization.Primitives" version="[4.1.1-rc3-24128-00, )" />
+        <dependency id="Newtonsoft.Json" version="[7.0.1, )" />
+        <dependency id="NETStandard.Library" version="[1.6.0-rc3-24128-00, )" exclude="runtime" />
```

Comment:
Why do we need this dependency if we are actually excluding the runtime? I understood why it was needed for the tool-runtime but I don't understand why it is needed here.